src/main/java/org/opensrp/web/rest/LocationResource.java

```diff
@@ -308,4 +310,34 @@ public void setLocationService(PhysicalLocationService locationService) {
 
 	}
 
+	static class LocationSyncRequestWrapper {
+		@JsonProperty
```

Comment:
why not pass the json attribute value on this tag so that the variables are named using camelCase convention e.g
`@JsonProperty("is_jurisdiction")`